Packs/Azure-Enrichment-Remediation/pack_metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Azure Enrichment and Remediation",
+    "description": "Playbooks using multiple Azure content packs for enrichment and remediation purposes",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "marketplaces": [
+        "xsoar",
+        "marketplacev2"
+    ],
+    "dependencies": {
+        "AzureCompute": {
+            "mandatory": true,
+            "display_name": "Azure Compute"
+        },
+        "CommonScripts": {
+            "mandatory": true,
+            "display_name": "Common Scripts"
+        },
+        "MicrosoftGraphUser": {
+            "mandatory": true,
+            "display_name": "Microsoft Graph User"
```

Comment:
Should it be mandatory?

Reply:
It should in this playbook, the all intent is to respond